### ISSUE TITLE
Does not work for me 

### ISSUE THREAD

**Issue by canufeel** [Blackening999/testing-menu#1]
ember-cli version: 0.2.7
node: 0.12.2
npm: 2.11.0
DEBUG: Ember      : 1.12.0
vendor.js:15442 DEBUG: Ember Data : 1.0.0-beta.18
vendor.js:15442 DEBUG: jQuery     : 1.11.3

just shows up Welcome to Ember.js and on home route it shows additional word - Home. The Welcome to Ember.js is all in gray color. What does this addon actually does?

**Comment by Blackening999** [comment]
Hello !

First of all, thanks for trying.
Secondly, do you have a button menu at the top left of the page?
Also it should work by tapping from bottom right border and sliding to left
- it's default sliding option

**Comment by canufeel** [comment]
Here is what i have got.

![screen shot 2015-06-04 at 12 35 20](https://cloud.githubusercontent.com/assets/5106750/7981254/5bee507c-0ab6-11e5-9ce1-57a80830ba3f.png)

**Comment by Blackening999** [comment]
Ok.

Well, it's strange since I don't have "Welcome to Ember.js" anywhere in the
project.

It might be the issue with latest ember-cli and ember. Since I've tested it
using :
DEBUG: Ember      : 1.8.1
Handlebars : 1.3.0
DEBUG: jQuery     : 1.11.3

Planning to upgrade it on this weekends.

I just pulled it from my own production project and there it isn't so easy
to make a transition to latest Ember you know :)

I'll let you know when it will be ready for latest Ember

Sorry for that inconvenience

чт, 4 июня 2015 г. в 13:37, canufeel notifications@github.com:
